includes/rest-api/class-wp-job-manager-data-stores-status.php

```diff
@@ -0,0 +1,69 @@
+<?php
+/**
+ * Declaration of our Status Data Store
+ *
+ * @package WPJM/REST
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Class WP_Job_Manager_Data_Stores_Status
+ */
+class WP_Job_Manager_Data_Stores_Status extends WPJM_REST_Data_Store_Abstract
```

Comment:
@dbtlr `WPJM_REST` is the unique prefix of Mixtape. `WP_Job_Manager` is the prefix of all our "own" classes and was there before mixtape. I would prefer to keep those separate, but ok with putting them on a similar prefix.

We could rename the Mixtape one to e.g. `WP_Job_Manager_REST` (if we do this, this will be done on a separate pr)